### DISCUSSION TITLE
(question) Multiple users 1 account?

### DISCUSSION THREAD

**Discussion by nssatlantis** [kimai/kimai#2927] [General]
Hello,

I am looking into a way to track work times for my employees, as we work with a 0 hour contract and they are essentially free to take a day off or work extra.

My only issue is that I want this to be as simple as possible, which had me hoping for the following;

Is it possible to have one 'main' user account and have them sign themselves in without having to log in and out per employee?
 At best I hope they can just hop onto the computer, activate themselves and not have to look back at it again. Is this a possibility, or is there any hacky way I could achieve this?

Thanks in advance!

**Comment by kevinpapst** [comment]
Have a look at https://www.kimai.org/store/kiosk-barcode-bundle.html

You could give them short & rememberable codes, which they use to clock-in and out.
